<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form3DReconstruction</class>
 <widget class="QDialog" name="Form3DReconstruction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_00">
       <item>
        <widget class="QPushButton" name="reconstruction_path_btn">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reconstruction_path"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_01">
       <item>
        <widget class="QLabel" name="reconstruction_savrpath_lab">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>保存名称</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reconstruction_savrpath">
         <property name="text">
          <string>reconstruction</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_02">
       <item>
        <widget class="QLabel" name="reconstruction_decimatepro_lab">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>压缩系数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="reconstruction_decimatepro">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="reconstruction_decimatepro_hor">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>96</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_03">
       <item>
        <widget class="QLabel" name="reconstruction_smooth_lab">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>平滑系数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="reconstruction_smooth">
         <property name="maximum">
          <number>3000</number>
         </property>
         <property name="singleStep">
          <number>0</number>
         </property>
         <property name="value">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="reconstruction_smooth_hor">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>96</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_04">
       <item>
        <widget class="QLabel" name="reconstruction_type_lab">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>图片类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reconstruction_type">
         <property name="text">
          <string>png</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="reconstruction_type_hor">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>96</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_05">
       <item>
        <widget class="QLabel" name="reconstruction_typename_lab">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>序列名称</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="reconstruction_typename">
         <property name="text">
          <string>%03d_mask</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="reconstruction_typename_hor">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>96</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_06">
       <item>
        <widget class="QLabel" name="reconstruction_sum_lab">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>图片起止编号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="reconstruction_sum_begin">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="reconstruction_sum_lab1">
         <property name="text">
          <string>——</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="reconstruction_sum_end">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="reconstruction_sum_hor">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>96</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_07">
       <item>
        <widget class="QLabel" name="reconstruction_spacing_lab">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>重构间隔(x:y:z)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="reconstruction_spacing_x">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="reconstruction_spacing_lab1">
         <property name="text">
          <string>：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="reconstruction_spacing_y">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="reconstruction_spacing_lab2">
         <property name="text">
          <string>：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="reconstruction_spacing_z">
         <property name="singleStep">
          <number>7</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="reconstruction_spacing_hor">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_08">
       <item>
        <widget class="QLabel" name="reconstruction_cubes_lab">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>等值面数值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="reconstruction_cubes">
         <property name="maximum">
          <number>3000</number>
         </property>
         <property name="singleStep">
          <number>0</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="reconstruction_cubes_hor">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>96</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="widget_reconstruction_buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>widget_reconstruction_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Form3DReconstruction</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget_reconstruction_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Form3DReconstruction</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
